source/core_http_client.c

```diff
@@ -1218,24 +1218,27 @@ static HTTPStatus_t addHeader( HTTPRequestHeaders_t * pRequestHeaders,
     {
         /* Write "<Field>: <Value> \r\n" to the headers buffer. */
 
-        /* Copy the header name into the buffer. */
-        ( void ) memcpy( pBufferCur, pField, fieldLen );
+        /* Copy the header name into the buffer. memcpy can be used here, but
```

Comment:
I would undo the change to the comment here as well as at lines 1226-1228.  You can use `strncpy()` without any additional explanation.

Reply:
Sure thanks